.platform/routes.yaml

```diff
@@ -115,6 +115,9 @@ https://{default}/:
       # Move CLI from Development to Administration in August 2022
       "/development/cli(.*)":  { "to": "/administration/cli$1", "code": 301, "regexp": true }
 
+      # Move project ownership transfer to orgs in August 2022
+      "/administration/users.html#transfer-project-ownership":  { "to": "/administration/organizations.html#transfer-project-ownership", "code": 301 }
```

Comment:
:lady_beetle:
The 301 doesn't seem to work on my end 🤔 
Opening:
https://move-that-org-pd7yzoa-652soceglkw4u.eu-3.platformsh.site/administration/users.html#transfer-project-ownership

does nothing

Reply:
That's why I'd removed the request for review before you posted this :laughing:

Reply:
Now I think I hadn't pushed correctly :thinking: